### DISCUSSION TITLE
Athos

### DISCUSSION THREAD

**Discussion by ssiwm** [gameserverapp/Platform#520] [Game request]
steam: https://store.steampowered.com/app/2526410/Athos/

dedicated server, info from developer discord: https://discord.com/channels/306480471779573780/1161398680545415168/1161398680545415168

Steps to setup a Athos Dedicated server on windows 10 only if you need for linux (ubuntu) a setup guide let me know

1 - Download steamcmd - (https://developer.valvesoftware.com/wiki/SteamCMD)

2 - click on wiindows then it will take you down in the page where it gives you the option for windows and linux

3 - click on download steamCMD for Windows: [1]

4 - next go to your download or anywhere you downloaded steamcmd and extract the steamcmd zip into a newfolder (does not matter what you call it the folder just don't have any spaces in the name) and make sure you move the folder to a harddrive that has space , also make sure you remember the location as this is where you will you access the server files

5 - after the extract is done click on the steamcmd.exe to install the steamcmd program and wait for it to finish

6 - as soon as its done you will see a steamcmd window so from here you can just type in - (login anonymous) then it should log you in to download the Athos server files
7 - Now when you logged in you can type (app_update 2577840) and then wait for the download to finish and as soon as the download is finish you can close steamcmd when it says Success! App '2577840' fully installed

8 - so now go to the location where you chose steamcmd to install , then go into the (steamapps/common/Athos Dedicated Server)

9 - so now you need to setup your server the way you like so you can so when you are in the athos dedicated server go look under (Athos Dedicated Server\Athos\Config) there is a DefaultGameSettings where you can change the server settings if you get stuck and not sure what to change and how to setup the Config you are more then welcome to comment on this forum and i am more then happy to help you as i wont go into to much detail regarding the config here unless its requested then ill edit the forum

10 - so after you are done with the config you can go back into the Athos Dedicated server folder that you are in and then just run AthosServer exe

OPTIONAL STEP BELOW WITH PHOTOS ATTACHED

there is no windows that pops up to see that the server is running so if you want to see a window to monitor the server

make a shortcut of AthosServer exe

and the right click on the shortcut and go to properties

and in the target bar after (") please add (-log) and click apply and OK

and then start the the server from the shortcut then the window will pop up to show the server is running 

so i hope this helps if you have any question please dont hesitate to ask

**Comment by mvaessen** [comment]
Thanks for the detailed request!
